climpred/smoothing.py

```
@@ -170,6 +170,8 @@ def temporal_smoothing(ds, tsmooth_kws=None, how="mean", d_lon_lat_kws=None):
             tsmooth_kws,
         )
     smooth = list(tsmooth_kws.values())[0]
+    if smooth == 1:
+        return ds
```

failed silently before

---

smooth == 1 doesnt make sense, no aggregation